gh-actions/common/validate-pr-title/action.yaml

```diff
@@ -0,0 +1,103 @@
+name: Validate pull request title
+description: Ensures the title of the pull request complies with naming policy
+
+outputs:
+  tag:
+    description: 'The tag parsed from the PR title'
+    value: ${{ steps.parse.outputs.tag }}
+  component:
+    description: 'The component parsed from the PR title'
+    value: ${{ steps.parse.outputs.component }}
+  title:
+    description: 'The title parsed from the PR title'
+    value: ${{ steps.parse.outputs.title }}
```

Comment:
I’m really unsure if there is any use of this as workflow output TBH. In my mind, generating the readme will be for each commits, when we cut a release, not everytime we merge something in main, otherwise, for each merge, you will get an additional update in main which will only create noise and a more complex history digging.

Reply:
Yeah, makes sense. I'll cut the output.